modules/apps/object/object-web/src/main/resources/META-INF/resources/object_entries/object_entry/ScheduleContainer.tsx

```diff
@@ -10,16 +10,20 @@ import ScheduleField from './ScheduleField';
 
 import './ScheduleContainer.scss';
 
-type SchedulePropertyKey = 'reviewDate';
+type SchedulePropertyKey = 'expirationDate' | 'reviewDate';
 
 interface SchedulePropertyValues {
 	checked: boolean;
 	value: string;
 }
 
+type ScheduleProperties = {
+	[key in SchedulePropertyKey]?: SchedulePropertyValues;
+};
```

Comment:
Hey @rtienpal, `[key in Schedule PropertyKey]` is not an optional property. We will always need to pass the `reviewDate` and `expirationDate` to the `scheduleContainer`. We can revert this modification and its implications.